install/kubernetes/cilium/values.yaml

```
@@ -863,6 +863,10 @@ prometheus:
     # -- Specify the Kubernetes namespace where Prometheus expects to find
     # service monitors configured.
     # namespace: ""
+  # Metrics that should be enabled or disabled from the default metric
```

Couple of things here:

nit, formatting like this (like the other options above) allows the automated tooling we use on these values to pick up the descriptions and properly format them:
```suggestion
  # -- Metrics that should be enabled or disabled from the default metric
```

Additionally, would it be reasonable to also document what the valid options are here? Or is that better documented somewhere else?

---

Added a reference to the docs.